cocos/3d/CCTerrain.h

```diff
@@ -424,7 +424,7 @@ class CC_DLL Terrain : public Node
     /**
      * get the terrain's size
      */
-    Size getTerrainSize() const { return Size(_imageWidth, _imageHeight); }
+    Size getTerrainSize() const { return Size(static_cast<float>(_imageWidth), static_cast<float>(_imageHeight)); }
```

Comment:
I think it is more reasonable to change the type of `_imageWidth ` and `_imageHeight ` to float. 
@ricardoquesada what do you think?